File-Formats/PDF/Convert-HTML-To-PDF/docker.md

```diff
@@ -7,91 +7,89 @@ documentation: UG
-     libnss3 libgbm1
+    RUN apt-get update && \
+        apt-get install -yq --no-install-recommends \ 
+        libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \ 
+        libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 \ 
+        libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 \ 
+        libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 \ 
+        libnss3 libgbm1
 
-{% endhighlight %}
+   {% endhighlight %}
 
-![Convert HTMLToPDF Docker Step4](htmlconversion_images/DockerStep4.png)
+   ![Convert HTMLToPDF Docker Step4](htmlconversion_images/DockerStep4.png)
 
-Add an Export to the PDF button in the index.cshtml.
+4. Add an Export to the PDF button in the index.cshtml.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-<div class="btn">
-    @{ Html.BeginForm("ExportToPDF", "Home", FormMethod.Post);
-        {
+   <div class="btn">
+      @{ Html.BeginForm("ExportToPDF", "Home", FormMethod.Post);
+         {
             <input type="submit" value="Export To PDF" class=" btn" />
-        }
-}
-</div>
-
-{% endhighlight %}
-
-![Convert HTMLToPDF Docker Step5](htmlconversion_images/DockerStep5.png)
-
-Include the following namespaces and code samples in the controller for converting HTML to PDF.
-
-{% highlight c# tabtitle="C#" %}
-
-using Syncfusion.HtmlConverter;
-using Syncfusion.Pdf;
-using System.IO
-
-{% endhighlight %}
-
-{% highlight c# tabtitle="C#" %}
-
-public ActionResult ExportToPDF()
-{
-     //Initialize HTML to PDF converter. 
-     HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
- 
-     BlinkConverterSettings settings = new BlinkConverterSettings();
-     
-     //Set command line arguments to run without the sandbox.
-     settings.CommandLineArguments.Add("--no-sandbox");
-     settings.CommandLineArguments.Add("--disable-setuid-sandbox");
-     
-     //Assign Blink settings to the HTML converter.
-     htmlConverter.ConverterSettings = settings;
- 
-     //Convert URL to PDF.
-     PdfDocument document = htmlConverter.Convert("https://www.syncfusion.com");
- 
-     MemoryStream stream = new MemoryStream();
- 
-     //Save and close a PDF document. 
-     document.Save(stream);
- 
-     return File(stream.ToArray(), System.Net.Mime.MediaTypeNames.Application.Pdf, "HTML-to-PDF.pdf");
-}
-
-{% endhighlight %}
-
-Build and run the sample in the docker. It will pull the Linux docker image from the docker hub and run the project. Now, the webpage will open in the browser. Click the Export to PDF option to convert the webpage to a PDF.
-
-By executing the program, you will get the PDF document as follows.
+         }
+      }
+   </div>
+
+   {% endhighlight %}
+
+   ![Convert HTMLToPDF Docker Step5](htmlconversion_images/DockerStep5.png)
+
+5. Include the following namespaces and code samples in the controller for converting HTML to PDF.
```

Comment:
Add separate step for add code snippet. So, 5th step will be "A default controller with name HomeController.cs gets added on creation of ASP.NET Core project. Include the following namespaces in that HomeController.cs file."

6. Add a new action method in HomeController.cs and include the below code snippet to convert HTML to PDF document.

Reply:
added separate step